app/src/main/java/ch/epfl/qedit/model/users/Administrator.java

```diff
@@ -0,0 +1,20 @@
+package ch.epfl.qedit.model.users;
+
+import java.util.ArrayList;
+import java.util.Date;
```

Comment:
We should use `Calendar` instead of `Date`, because the documentation says it is deprecated...

Reply:
Ok I'll change this